parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -390,6 +390,21 @@ ensureKubelet() {
     {{end}}
 }
 
+# The labels.service on startup and every 5 minutes periodically ensures that the customnodelabels 
+# are applied to the node.
+ensureLabels() {
```

Comment:
I think we should combine this into the existing label-nodes service

1) they're doing two steps but for the same purpose to ensuring the nodes' label are good. separating them doesn't make sense to me
2) it causes this unnecessary naming confusion between ensureLabels and ensureLabelNodes
3) duplicated label-nodes.sh file for the same purpose

Reply:
ok, on a bit closer look, looks like ensureLabelNodes is a legacy we inherit from aks-e code but it was not really being used and should be deleted. The corresponding work has been moved to CCP side in AKS RP (if you search by the content from the script in AKS RP you will see). 

Can you help remove it to avoid future confusion? Thanks.

Reply:
I'm happy to help remove it. However, I am leaning towards doing it in an additional PR, so this does not be cluttered with the change ripping out the label-nodes code